### PR TITLE
Basic Updates

### PR DESCRIPTION
**Pull request by christiantonti**
-Added loading spinner while awaiting Heroku response (there is a 20-30 second API delay after Heroku gets no requests for 30 minutes)
-Centered table header text
-Added title bar and short description
-Added general padding and some bootstrap styling

Further updates to come; maybe coloration, a user lookup, or a login screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,9 +11,18 @@
...
+      <h1 class="text-center">Kappa Theta Pi Strikes</h1>
+      <h5 class="text-center">A log of all strikes added and removed and their reasons why.</h5>  
+    </div>
+    <div id="loading-message" class="text-center mt-2">
+      <h5 class="py-2">Loading Table...</h5>
+      <div class="spinner-border text-primary" role="status">
+        <span class="sr-only">Loading...</span>
+      </div>  
+    </div>
+    <table id="table" class="table table-striped" style="display: none;">
```
Comment (evinaik) [inline index.html]: Why is this style inline?

File: index.html [hunk 2]
```diff
@@ -11,9 +11,18 @@
...
   <body>
-
-    <table id="table">
-      <thead>
+    <div id="title" class="py-3">
+      <h1 class="text-center">Kappa Theta Pi Strikes</h1>
+      <h5 class="text-center">A log of all strikes added and removed and their reasons why.</h5>  
+    </div>
+    <div id="loading-message" class="text-center mt-2">
+      <h5 class="py-2">Loading Table...</h5>
```
Comment (evinaik) [inline index.html]: Can you provide some context about these py-X classes? I'm not sure what they are, though I'm interested.

**Comment by evinaik** [comment]
Great speed on this. For UI PR's, you generally include a before/after screenshot (just after is okay), for a sanity check. Once you add that and comments, I'll approve.

**Review by evinaik — CHANGES_REQUESTED** [review]
See comments.

File: index.html (hunk 1 above)
Comment (christiantonti) [reply index.html]: Using the bootstrap display none class resulted in the table not appearing upon using jQuery's fadeIn() function, so an inline worked best. Obviously a style tag in the header would work as well but I maximized bootstrap use.

File: index.html (hunk 2 above)
Comment (christiantonti) [reply index.html]: Bootstrap has spacing classes for margin and padding. 'p' stands for padding, 'y' stands for top and bottom (y-axis), and the number is a multiplier to the $spacer bootstrap sass variable.
See: https://getbootstrap.com/docs/4.3/utilities/spacing/

**Comment by christiantonti** [comment]
Changes and a UI before/after photo will be up by tonight, along with some comments in the code for clarity.

**Comment by christiantonti** [comment]
Added descriptive comments to some areas for clarity. Also moved inline styles to the <style> tag in the head. Attached are a before and after photo of the UI.
Before:
image
After (note that the loading spinner is not shown here):
image

**Comment by evinaik** [comment]
LGTM

**Pull request merged**